website/source/docs/agent/options.html.md

```diff
@@ -1449,6 +1449,17 @@ Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'."
         single RPC call to a Consul server. See
         https://en.wikipedia.org/wiki/Token_bucket for more details about how
         token bucket rate limiters operate.
+    *   <a name="kv_max_value_size"></a><a href="#kv_max_value_size">
+        `kv_max_value_size`</a> - Configures the maximum number of
+        bytes for a kv request body to the [`/v1/kv`](/api/kv.html)
+        endpoint. This limit defaults to [raft's](https://github.com/hashicorp/raft)
+        suggested max size. **Note that increasing beyond this default can
```

Comment:
Maybe this note is a little strong considering there are many cases where we tell users to increase this.

For the gossip tunables we say "Tuning these improperly can cause Consul to fail in unexpected ways."

I think something like that could be more appropriate because we have several users using increased values without problems and secondly decreasing this value too low could also cause seemingly legitimate requests to fail. For those reasons I like the existing language in other places that says tuning them "improperly" could cause weird failures.

Reply:
Agreed!